crates/router/src/core/webhooks/webhook_events.rs

```diff
@@ -1,4 +1,7 @@
-use error_stack::ResultExt;
+use std::collections::HashSet;
+
+use common_enums::{EventClass, EventType};
```

Comment:
Please avoid importing types (enums in this case) directly, qualify their usages instead.